setup.cfg

```diff
@@ -145,7 +145,7 @@ cylc.command =
     diff = cylc.flow.scripts.diff:main
     dump = cylc.flow.scripts.dump:main
     ext-trigger = cylc.flow.scripts.ext_trigger:main
-    extract-resources = cylc.flow.scripts.extract_resources:main
+    resources = cylc.flow.scripts.extract_resources:main
```

Comment:
Has this rename been discussed? Personally `extract-resources` is clearer to me

Reply:
between @oliver-sanders  and I. Perhaps this should be thrown open as a discussion. 
Would `get-resouces` or `get-files` do for you?

Reply:
`get-resources` seems fine to me. I was just curious about the reason for the change

Reply:
make it shorter since users will be typing it to get the tutorials.

Reply:
Open to offers, IMO `get-resources` is too long:

Context:
* `extract-resources` is a bit of a mouthful and not especially obvious anyway.
* This command will probably be used to extract the Cylc tutorial so we want something convenient to type.
* The rose equivalent is called `rose resource`.

Reply:
I was going for a plural, because by default the script retrieves everything.
Is that desirable behavior if we are listing all the tutorials?

Given the somewhat different users (cylc extract-resources in its current form is an internal/site admin command, whilst tutorial is explicitly for newbies) might it  be sensible to have a  `cylc tutorial` command. This would in effect be an alias, with slightly differing behavior (i.e. a different list of resources) depending on `sys.argv[2]`.

Reply:
Alternative name suggestions?

Reply:
If it's just a command that most users will only use when first learning about Cylc and doing the tutorials, I feel it's ok to have a bit of a long command name

Reply:
I'm going to make a decision and go for `cylc get-resources`. 
Reasoning:
- 95% of users will be copy and pasting it out of tutorials anyway.
- Get < Extract
- reading_age('get') < reading_age('extract') for ESL users.
- `resources` alone not massively obvious.
- `resources` because it's designed to extract 1+ resources.

The only negative I can see is that it's inconsistent with `rose resource`, but since I think most people using it will either be copying and pasting from the tutorial or expert users I'm not too worried about this. If it becomes a problem we can always put an alias in.